contents/handbook/engineering/release-new-version.md

```diff
@@ -75,11 +82,11 @@ The release manager is ultimately responsible for the timeline of the release. T
     1. If the `helm upgrade` command fails or if in the end the output for `kubectl get pods -n posthog` doesn't show everything as running, then ask `team-platform` for guidance.
     1. Optional: Verify playground is running the latest image by running `kubectl get pod --namespace posthog`. In the output of that command, you should see a row like `posthog-web-6447ff5fdf-gs664`. Copy this row (the numbers after `posthog-web-` will be different), and then run `kubectl  describe pod --namespace posthog posthog-web-6447ff5fdf-gs664`. If you scroll up in that output, you should see a line like `Image: posthog/posthog@sha256:daf43a4a4cd06658e41273bb8fe4a74f17b295d67c6f1e16c17243b5d09af7ee`. This is the sha of the image that is running. You can compare this to the sha in Docker Hub to verify that the image is the latest.
     1. Go to the [playground](https://playground.posthog.net/) and test that everything is working as expected. Check that the version running is the same as the one we're releasing.
-1. [ ] Time for the "Break the release" session! It's imperative that the session uses the published `release-[version]-unstable` image from Docker Hub to avoid any potential bugs creeping up in the final build stage. You're responsible for running the session, prepare the [release checklist doc](https://docs.google.com/document/d/1tyTChgLM8ZKCIyP05yDeyzS7y-s2ii5lm5WltREO3so) by adding the template at the top.
-1. [ ] Figure out what's updated in this release with the command below or by asking the Product or Engineering Team. The command will output the entire commit list to `changelog.txt`, sorted by PR [type and scope](https://www.conventionalcommits.org/en/). You can use this list to obtain external contributions to highlight in the Array. In addition, you can look for the `highlight` tag in PRs but be mindful it's not used very consistently.
+1. [ ] Time for the "Break the release" session! It's imperative that the session uses the published `release-[version]-unstable` image from Docker Hub to avoid any potential bugs creeping up in the final build stage. You're responsible for running the session, prepare the [release checklist doc](https://docs.google.com/document/d/1tyTChgLM8ZKCIyP05yDeyzS7y-s2ii5lm5WltREO3so) by adding the template at the top. Note that you're also responsible for making sure everything got testes and getting cherry picking the fixes and prs tagged with `release-<version>` into the release branch.
```

Comment:
```suggestion
1. [ ] Time for the "Break the release" session! It's imperative that the session uses the published `release-[version]-unstable` image from Docker Hub to avoid any potential bugs creeping up in the final build stage. You're responsible for running the session, prepare the [release checklist doc](https://docs.google.com/document/d/1tyTChgLM8ZKCIyP05yDeyzS7y-s2ii5lm5WltREO3so) by adding the template at the top. Note that you're also responsible for making sure everything is tested and for cherry picking the fixes and prs tagged with `release-<version>` into the release branch.
```